css/aui-components/dropdown.scss

```diff
@@ -0,0 +1,46 @@
+form.aui {
+    .aui-select2-container {
+        background-color: transparent;
+
+        > a{
+            margin: 0 auto !important;
+            width: 100% !important;
+        }
+    }
+
+}
+.select2-results{
+    &::-webkit-scrollbar {
+        background-color: transparent;
+    }
+    &::-webkit-scrollbar-track {
+        background-color: transparent;
+    }
+    &::-webkit-scrollbar-track-piece {
+        background-color: transparent;
+    }
```

Comment:
@kkrisstoff, why are `&::-webkit-scrollbar`, `&::-webkit-scrollbar-track`, and `&::-webkit-scrollbar-track-piece` the same i.e. `background-color: transparent;` and yet written separately? Is that out of a convenience for future editing should there arise a need to have them defined differently?